src/components/autocomplete/autocomplete.tsx

```diff
@@ -25,6 +25,7 @@ export interface AutocompleteProps {
     autoCompleteRef?: (api: AutocompleteApi) => any;
     filterSuggestions?: boolean;
     qeid?: string;
+    autoHighlight?: boolean;
```

Comment:
```suggestion
    autoHighlight?: true;
```

Let's keep the same behavior for backward compat

Reply:
```suggestion
    autoHighlight?: true; // ReactAutocomplete.Props['autoHighlight'] is missing the `true` default per https://github.com/argoproj/argo-ui/pull/560#issuecomment-2155872347
```

including my [below](https://github.com/argoproj/argo-ui/pull/560#issuecomment-2155872347) suggestion with a comment

Reply:
Follow-up [thread], apparently interface defaults are unsupported, so will just use JSDoc `@default` annotation